src/dotnet/Program.cs

```diff
@@ -56,6 +57,8 @@ public static int Main(string[] args)
             {
                 using (PerfTrace.Current.CaptureTiming())
                 {
+                    ConfigureDotNetForFirstTimeUse();
```

Comment:
Should this be inside it's own PerfTrace? So we can understand it's individual perf characteristics